tests/foreman/ui/test_host.py

```diff
@@ -2097,7 +2097,7 @@ def test_bootc_booted_container_images(target_sat, bootc_host, function_ak_with_
 
     :Verifies:SAT-27163
 
-    :Team: Phoenix-subscriptions
+    :Team: Artemis
```

Comment:
bootc is covered by Artmis, @vsedmik should this be on Artemis or Proton?

Reply:
I would prefer to keep it on Artemis as SME for `bootc`.